static/css/main.css

```diff
@@ -6,3 +6,69 @@
 .link-success a {
   color: white;
 }
+
+table.dataTable thead>tr>td.sorting,
```

Comment:
Where did you get all of these lines?

`main.css` is for custom stuff, but this seems related to the `DataTable` plugin

Reply:
https://mdbootstrap.com/docs/jquery/tables/scroll/